### DISCUSSION TITLE
What is an example of a workflow that generates an attribute_name in a node?

### DISCUSSION THREAD

**Discussion by scottwn** [AgnostiqHQ/covalent#789] [Q&A] [answered]
In the [database schema](https://drive.google.com/drive/folders/1HIkM7Dr_cgQpas_B00JCIkixGaertH7T), there is an `attribute_name` field in the electrons table. It is described as "attribute name for attribute nodes", but I'm not sure how to create an attribute node. 

[I can see where this value is set in the node](https://github.com/AgnostiqHQ/covalent/blob/develop/covalent/_workflow/electron.py#L273), but I'm not sure how to get this block to execute. I previously tried to write [tests](https://github.com/AgnostiqHQ/covalent/issues/705) for this block (it is uncovered), but struggled there as well.

Could someone share an example workflow that generates this type of node?

Loosely related to #760

**Comment by kessler-frost** [comment, marked as answer]
Example workflow for attribute node:

```python
import covalent as ct

class Test:
    def __init__(self):
        self.exampling = 210

@ct.electron
def task_1():
    return Test()

@ct.electron
def task_2(x):
    return x * 2

@ct.lattice
def workflow():

    res_1 = task_1()
    res_2 = task_2(res_1.exampling)

    return res_2

result = ct.dispatch_sync(workflow)()
print(result)
```

**Reply by kessler-frost** [reply]
@scottwn I'm marking this as an answer but please feel free to change that if you think this doesn't answer your question.
